test/org/sagebionetworks/bridge/play/controllers/SurveyControllerTest.java

```diff
@@ -132,11 +144,18 @@ private void setUserSession(String studyIdentifier) {
                 .withHealthCode("BBB")
                 .withRoles(Sets.newHashSet(DEVELOPER)).build();
         // The mock user is initially
+        SubpopulationGuid guid = SubpopulationGuid.create("subpopGuid");
+        ConsentStatus status = new ConsentStatus.Builder().withName("Name").withConsented(true).withGuid(guid)
+                .withSignedMostRecentConsent(true).build();
+        
+        Map<SubpopulationGuid,ConsentStatus> consentStatuses = Maps.newHashMap();
+        consentStatuses.put(guid, status);
+        
         session = new UserSession(participant);
         session.setStudyIdentifier(new StudyIdentifierImpl(studyIdentifier));
-        doReturn(session).when(controller).getAuthenticatedSession();
-        doReturn(session).when(controller).getAuthenticatedAndConsentedSession();
-        doReturn(session).when(controller).getAuthenticatedSession(any(Roles.class));
+        session.setConsentStatuses(consentStatuses);
```

Comment:
why we need to set consent statuses up while we didn't use it in previous test codes?